packages/@coorpacademy-components/src/molecule/external-content-button/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Provider from '../../atom/provider';
+import {EXTERNAL_CONTENT_ICONS} from '../../util/external-content';
+
+import style from './style.css';
+
+class ExternalContentButton extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+    this.handleClick = this.handleClick.bind(this);
```

Comment:
No need to `bind` when using arrrow functions

Reply:
bien vu